src/MatchMaker/Encounter/Encounter.php

```diff
@@ -20,8 +20,8 @@ class Encounter
     public const STATUS_PLAYING = 'En cours';
     public const STATUS_OVER = 'Terminé';
 
-    public int $scorePlayerA;
-    public int $scorePlayerB;
+    private mixed $scorePlayerA = NULL;
+    private mixed $scorePlayerB = NULL;
```

Comment:
`mixed` is rarely a good idea unless you are creating a really abstract code for a framework or an open source module. it is way too permissive. keeping things closer to what you expect will avoid weird types to be set by mistake.

```suggestion
    private ?int $scorePlayerA = null;
    private ?int $scorePlayerB = null;
```